1-js/02-first-steps/14-function-basics/article.md

```diff
@@ -332,6 +332,26 @@ return*!*;*/!*
  (some + long + expression + or + whatever * f(a) + f(b))
 ```
 So, it effectively becomes an empty return. We should put the value on the same line instead.
+
```

Comment:
For spreading an expression into multiple lines - a good idea to say that, but need to add rules when it works. 

Not only "+" operator can do that.

Probably, more details about that should go to another part of the tutorial where semicolons are discussed.